core/domain/exp_domain.py

```diff
@@ -395,6 +404,8 @@ def __init__(
                 enabled.
             correctness_feedback_enabled: bool. True if correctness feedback is
                 enabled.
+            image_counter: int. The counter that is responsible for creating
+                image ids. Image Id is always <= image_counter.
```

Comment:
```suggestion
                image IDs. Image ID is always less than or equal to image_counter.
```
I initially though <= was some form of arrow :sweat_smile: